rtengine/procparams.cc

```diff
@@ -1884,6 +1884,46 @@ bool SHParams::operator !=(const SHParams& other) const
     return !(*this == other);
 }
 
+
+
+CGParams::CGParams() :
+    enabled(false),
+    th_c(0.815),
+    th_m(0.803),
+    th_y(0.880),
+    d_c(1.147),
+    d_m(1.264),
+    d_y(1.312),
+    pwr(1.2),
+    colorspace("srgb"),
```

Comment:
Why not ? But in most cases most users' screens (like mine) are sRGB. but I don't see any drawbacks to that.

Reply:
The thresholds and limits are optimized for ACES AP1. If we use sRGB for the default, then different thresholds and limits will give better results. At least, we should have different thresholds because they are computed to protect the colors of the 24 patch color checker. I am not too concerned about the limits because they are based on the color spaces of cinema cameras.